packages/examples/aws-codebuild/src/CI.Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:12-slim
```

Comment:
I copied the image inside this repository. This makes it easier to follow whats going on.